src/main/java/com/code_intelligence/jazzer/driver/Opt.java

```diff
@@ -147,8 +147,8 @@ public final class Opt {
       boolSetting("internal.conditional_hooks", false, null);
 
   // Some scenarios require instrumenting the jar before fuzzing begins
-  public static final boolean instrumentOnly =
-      boolSetting("instrument_only", false, "Instrument jar file. No fuzzing is performed.");
+  public static final List<String> instrumentOnly =
+      stringListSetting("instrument_only", ',', "Instrument jar file. No fuzzing is performed.");
```

Comment:
Mention the separator char in the description? Something like `comma separated list of ...`?